tests/acceptance/features/lib/UsersPage.php

```diff
@@ -658,6 +661,42 @@ public function changeUserPassword(Session $session, $user, $password) {
 		}
 	}
 
+	/**
+	 * @param Session $session
+	 * @param string $user
+	 * @param string $password
+	 *
+	 * @return void
+	 */
+	public function changeUserEmail(Session $session, $user, $password) {
+		$userTr = $this->findUserInTable($user);
+		$editEmailBtn = $userTr->find("xpath", $this->editEmailBtnXpath);
+		if ($editEmailBtn === null) {
+			throw new ElementNotFoundException(
+				__METHOD__ .
+				" xpath $this->editEmailBtnXpath " .
+				"could not find edit button "
+			);
+		}
+		$editEmailBtn->focus();
+		$editEmailBtn->click();
+		$this->waitForAjaxCallsToStartAndFinish($session);
+		$editEmailInput = $userTr->find("xpath", $this->editEmailInputXpath);
+		if ($editEmailInput === null) {
+			throw new ElementNotFoundException(
+				__METHOD__ .
+				" xpath $this->editEmailInputXpath " .
+				"could not find email field "
+			);
+		}
+		try {
+			$editEmailInput->focus();
+			$editEmailInput->setValue($password . "\n");
+			$this->waitForAjaxCallsToStartAndFinish($session);
+		} catch (StaleElementReference $e) {
```

Comment:
Please explain why this catch is required here. Might help someone in the future.